drivers/sensors/pimoroni_trackball.c

```diff
@@ -16,7 +16,9 @@
  */
 #include "pimoroni_trackball.h"
 #include "i2c_master.h"
+#ifdef TRACKBALL_DEBUG
 #include "print.h"
+#endif
```

Comment:
Given the nop stubs <https://github.com/qmk/qmk_firmware/blob/master/quantum/logging/print.h#L52-L59>, there should be no need for this change.

```suggestion
#include "print.h"
```